src/js_command_match/add_credentials.rs

```diff
@@ -0,0 +1,97 @@
+const DEFAULT_SEED_PHRASE_PATH: &str = "m/44'/397'/0'";
+
+#[derive(Debug, Clone, clap::Parser)]
+/// This is a legacy `legacy` command. Once you run it with the specified arguments, new syntax command will be suggested.
+pub struct AddCredentialsArgs {
+    account_id: String,
+    #[clap(long, aliases = ["seedPhrase"], default_value = None, conflicts_with = "secret_key")]
+    seed_phrase: Option<String>,
+    #[clap(long, aliases = ["secretKey"], default_value = None)]
+    secret_key: Option<String>,
+    #[clap(long, aliases = ["networkId"], default_value=None)]
+    network_id: Option<String>,
+    #[clap(allow_hyphen_values = true, num_args = 0..)]
+    _unknown_args: Vec<String>,
+}
+
+impl AddCredentialsArgs {
+    pub fn to_cli_args(&self, network_config: String) -> Vec<String> {
+        let network_id = self.network_id.clone().unwrap_or(network_config.to_owned());
```

Comment:
This `.to_owned()` is unnecessary as it is already an owned `String` (not a reference):

```suggestion
        let network_id = self.network_id.clone().unwrap_or(network_config);
```